lib/core/InterceptorManager.js

```diff
@@ -15,11 +14,12 @@ function InterceptorManager() {
  * @return {Number} An ID used to remove interceptor later
  */
 InterceptorManager.prototype.use = function use(fulfilled, rejected) {
-  this.handlers.push({
+  this.handlers[this.count] = {
     fulfilled: fulfilled,
     rejected: rejected
-  });
-  return this.handlers.length - 1;
+  };
+  this.count++;
```

Comment:
Pretty sure you can drop the custom count handling and just use Object.keys(this.handlers).length - let me know if this works!